projects/lib/src/lib/services/resource/resource.service.ts

```diff
@@ -64,14 +67,15 @@ export class ResourceService {
     fieldsOrRawQuery: any[] | string,
     kind: string,
     resourceId: string,
+    namespace: string,
     operation: string,
   ) {
     if (fieldsOrRawQuery instanceof Array) {
       return (
         gqlBuilder
           .query({
             operation: kind,
-            variables: { name: { value: resourceId, type: 'String!' } },
+            variables: { name: { value: resourceId, type: 'String!' }, namespace: { value: namespace, type: 'String' } },
```

Comment:
shouldn't the namespace variable be optional depending if we have a namespaced resource?